docs/excel/excel-add-ins-pivots.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Work with PivotTables using the Excel JavaScript API
+description: ''
+ms.date: 7/11/2018
+---
+
+
+
+# Work with PivotTables using the Excel JavaScript API
+
+PivotTables streamline larger data sets. They allow the quick manipulation of grouped data. The Excel JavaScript API lets your add-in create PivotTables and interact with their components. 
+
+If you are unfamiliar with the functionality of PivotTables, consider exploring them as an end-user. 
+See [Create a PivotTable to analyze worksheet data](https://support.office.com/en-us/article/Import-and-analyze-data-ccd3c4a6-272f-4c97-afbb-d3f27407fcde#ID0EAABAAA=PivotTables) for a good primer on these tools. 
+
+This article provides code samples for common scenarios.
+For the complete list of properties and methods the **PivotTable** and **PivotTableCollection** objects support, see [PivotTable API](https://dev.office.com/reference/add-ins/excel/pivottable) and [PivotTable Collection API](https://dev.office.com/reference/add-ins/excel/pivottablecollection).
+
+> [!NOTE]
+> These samples use APIs currently available only in public preview (beta). To run these samples, you must use the beta library of the Office.js CDN: https://appsforoffice.microsoft.com/lib/beta/hosted/office.js.
```

Comment:
lets make sure to add a link on how to get insiders fast and also refer to build 16.0.10801.20004 as a build someone can try the API with.